### ISSUE TITLE
Issues with the i18n 

### ISSUE THREAD

**Issue by boboci9** [aslagle/reactive-table#288]
Hi,
It's possible that it is the same issue as https://github.com/aslagle/reactive-table/issues/50
but I have a different error:

![screenshot from 2015-09-19 11 37 40](https://cloud.githubusercontent.com/assets/9323474/9975372/015408bc-5ec3-11e5-9c43-9d6e1a071429.png)

Has there been any solutions on this issue?

**Comment by aslagle** [comment]
The problem in #50 was a conflict with another package, and it was fixed. It's possible you have a different conflict - what other packages are you using?

**Comment by boboci9** [comment]
Hi, I am using the following packages:
```
# Core Meteor Packages
meteor-platform
oauth-encryption
underscore
less

# Optional Meteor Packages
accounts-facebook
spiderable
audit-argument-checks

# Reaction Commerce Packages
reactioncommerce:core
reactioncommerce:bootstrap-theme
reactioncommerce:reaction-accounts
reactioncommerce:reaction-shipping
reactioncommerce:reaction-paypal
reactioncommerce:reaction-braintree
reactioncommerce:reaction-stripe
reactioncommerce:reaction-auth-net
reactioncommerce:reaction-social
reactioncommerce:reaction-analytics-libs
reactioncommerce:reaction-analytics

# Community Packages
nemo64:bootstrap
```
Thanks you for your answer.

**Comment by aslagle** [comment]
Yeah, it looks like this is a conflict with Reaction Commerce. The reactioncommerce:core package creates a template helper called i18n, which conflicts with anti:i18n's template helper that's used in reactive-table. I'm not sure where it would be best to fix this - @aaronjudd any thoughts?

**Comment by aaronjudd** [comment]
The actually i18n methods are probably pretty similar - maybe I can make them compatible.  If not, I guess we'll either need to change the Reaction methods to have a unique name,  or customize reactive-table to let us define what i18n helpers to use.  I'll take a look today.
